thunder-service/service/oci_volume.go

```diff
@@ -60,10 +60,22 @@ func (as *ThunderService) GetOciUnusedStorage(profiles []string) ([]model.OciErc
 	}
 
 	for _, vl := range volumeList {
+		recommendation.Type = model.RecommendationTypeUnusedStorage
 		recommendation.CompartmentID = vl.CompartmentID
 		recommendation.CompartmentName = vl.CompartmentName
 		recommendation.ResourceID = vl.ResourceID
 		recommendation.Name = vl.Name
+		recommendation.ObjectType = model.ObjectTypeBlockStorage
+		detail1 := model.RecDetail{Name: "Block Storage Name", Value: vl.Name}
+		detail2 := model.RecDetail{Name: "Size", Value: vl.Size}
+		detail3 := model.RecDetail{Name: "Vpu", Value: vl.VpusPerGB}
+		detail4 := model.RecDetail{Name: "Attached", Value: "No"}
+
+		recommendation.Details = append(recommendation.Details, detail1)
```

Comment:
You can append multiple strings in the same `append` call.
https://go.dev/play/p/Dkdnvcsx4Q1
Would you mind to update?

Reply:
Done